src/services/db/index.ts

```diff
@@ -1,17 +1,22 @@
-import {Connection, createConnection, getConnectionOptions} from "typeorm";
+import {Connection, createConnection, getConnectionOptions, ObjectType, getConnection} from "typeorm";
 import {sleep} from "../utils";
 import {logger} from "../logger";
 import {PostgresConnectionCredentialsOptions} from "typeorm/driver/postgres/PostgresConnectionCredentialsOptions";
 import {TypeOrmLogger} from "../logger/typeorm";
 
 export async function getDatabaseConnection(): Promise<Connection> {
   const opts = await getConnectionOptions();
-  let conn = await createConnection({
-    ...opts,
-    logger: new TypeOrmLogger()
-  });
-  conn = await openConnection(conn);
-  await conn.runMigrations({transaction: "all"});
+  let conn: Connection;
+  try {
+    conn = getConnection();
+  } catch{
+    conn = await createConnection({
+      ...opts,
+      logger: new TypeOrmLogger()
+    });
+    conn = await openConnection(conn);
+    await conn.runMigrations({transaction: "all"});
```

Comment:
Don't think we should run migrations in catch clause.
Maybe move it to app initialization or something